addons/pos_restaurant/models/pos_order.py

```diff
@@ -259,3 +260,8 @@ def _export_for_ui(self, order):
         result = super(PosOrder, self)._export_for_ui(order)
         result['table_id'] = order.table_id.id
         return result
+
+    def get_all_table_draft_orders(self, pos_config_id):
```

Comment:
I forgot about this
```suggestion
    @api.model
    def get_all_table_draft_orders(self, pos_config_id):
```
Since `self` isn't a recordset relevant

Reply:
Done